text_generation/causal_lm/cpp/CMakeLists.txt

```
@@ -28,3 +28,11 @@ find_package(OpenVINO REQUIRED COMPONENTS Runtime)
 target_link_libraries(speculative_decoding_lm PRIVATE openvino::runtime)
 set_target_properties(speculative_decoding_lm PROPERTIES CXX_STANDARD 17)
 set_target_properties(speculative_decoding_lm PROPERTIES CXX_STANDARD_REQUIRED ON)
+
+add_executable(prompt_lookup_decoding prompt_lookup_decoding.cpp)
```

```suggestion
add_executable(prompt_lookup_decoding_lm prompt_lookup_decoding.cpp)
```
Follow the style of `speculative_decoding_lm` name

---

done